### PR TITLE
systemd-nspawn: add page

### PR DESCRIPTION
**Pull request by quantumflo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-nspawn.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# systemd-nspawn
+
+> Spawn a command or OS in a light-weight container.
+> More information: <https://www.freedesktop.org/software/systemd/man/latest/systemd-nspawn.html>.
+
+- Run a command in a container:
+
+`systemd-nspawn -D {{/path/to/container/root}}`
```
Comment (gutjuri) [inline pages/linux/systemd-nspawn.md:8]:
```suggestion
`systemd-nspawn -D {{path/to/container/root}}`
```

File: pages/linux/systemd-nspawn.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Run a full Linux-based OS in a container:
+
+`systemd-nspawn --boot -D {{/path/to/container/root}}`
```
Comment (gutjuri) [inline pages/linux/systemd-nspawn.md:12]:
```suggestion
`systemd-nspawn --boot -D {{path/to/container/root}}`
```

File: pages/linux/systemd-nspawn.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- By default, the command you specify is run as PID 1 (init) in the container. To run it as PID 2 with a stub init process:
+
+`systemd-nspawn -D {{/path/to/container/root}} --as-pid2`
```
Comment (gutjuri) [inline pages/linux/systemd-nspawn.md:16]:
```suggestion
`systemd-nspawn -D {{path/to/container/root}} --as-pid2`
```

File: pages/linux/systemd-nspawn.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Specify the machine name and hostname:
+
+`systemd-nspawn --machine="{{mycontainer}}" --hostname="{{mycontainerhost}}" -D {{/path/to/container/root}}`
```
Comment (gutjuri) [inline pages/linux/systemd-nspawn.md:20]:
```suggestion
`systemd-nspawn --machine={{mycontainer}} --hostname={{mycontainerhost}} -D {{path/to/container/root}}`
```

File: pages/linux/systemd-nspawn.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Run a command in a container:
+
+`systemd-nspawn -D {{/path/to/container/root}}`
+
+- Run a full Linux-based OS in a container:
+
+`systemd-nspawn --boot -D {{/path/to/container/root}}`
+
+- By default, the command you specify is run as PID 1 (init) in the container. To run it as PID 2 with a stub init process:
```
Comment (gutjuri) [inline pages/linux/systemd-nspawn.md:14]:
```suggestion
- Run the specified command as PID 2 in the container (as opposed to PID 1). Use a stub init process:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have added some suggestions :)

**Review by gutjuri — APPROVED** [review]
Thanks!

File: pages/linux/systemd-nspawn.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`systemd-nspawn --boot -D {{path/to/container/root}}`
+
+- Run the specified command as PID 2 in the container (as opposed to PID 1). Use a stub init process:
+
+`systemd-nspawn -D {{path/to/container/root}} --as-pid2`
+
+- Specify the machine name and hostname:
+
+`systemd-nspawn --machine={{mycontainer}} --hostname={{mycontainerhost}} -D {{path/to/container/root}}`
```
Comment (kbdharun) [inline pages/linux/systemd-nspawn.md:20]:
```suggestion
`systemd-nspawn --directory {{path/to/container_root}}`

- Run a full Linux-based OS in a container:

`systemd-nspawn --boot --directory {{path/to/container_root}}`

- Run the specified command as PID 2 in the container (as opposed to PID 1). Use a stub init process:

`systemd-nspawn --directory {{path/to/container_root}} --as-pid2`

- Specify the machine name and hostname:

`systemd-nspawn --machine={{mycontainer}} --hostname={{mycontainerhost}} --directory {{path/to/container_root}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Note:**

- We generally prefer long options and use short options when it is the only one available or the long option is highlighted via a mnemonic (`[e]xample`). In this case, `--directory` would be better to understand than `-D`.

File: pages/linux/systemd-tmp-files.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Clean up files and directories with age parameters configured:
+
+`systemd-tmpfiles --clean`
+
+- Remove files and directories as specified in the configuration:
+
+`systemd-tmpfiles --remove`
+
+- Apply operations for user-specific configurations::
```
Comment (kant) [inline pages/linux/systemd-tmp-files.md]: final `::`

**Comment by kbdharun** [comment]
I think this PRs branch is beyond recovery unfortunately, I will port the page from here to a new PR retaining the author's details.

**Comment by acuteenvy** [comment]
What's happening? There's like 5 different PRs with completely irrelevant commits right now.

**Comment by kbdharun** [comment]
> What's happening? There's like 5 different PRs with completely irrelevant commits right now.

I have fixed the other PRs. Sorry for causing the issue. 

What happened was @sebastiaanspeck used GitHub mobile to merge the PRs which seemed to have done rebase & merge and also bypassed our commit title defaults (for a couple of commits in the past few days). I fixed it in a force push for commits made in the last 4 days (after bypassing the protection rule). [We generally don't do force pushes at all (in a majority of cases) for exactly this purpose. But this time the issue was worse as Codeowners were requested reviews too.

**Pull request closed without merging**